ui/pages/settings/security-tab/reveal-srp-list/reveal-srp-list.test.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import configureStore, { MetaMaskReduxState } from '../../../../store/store';
+import mockState from '../../../../../test/data/mock-state.json';
+import { renderWithProvider } from '../../../../../test/lib/render-helpers';
+import { RevealSrpList } from './reveal-srp-list';
+
+const mockKeyringId = 'hd-keyring-01JKAF3DSGM3AB87EM9N0K41AJ';
```

Comment:
IMO, that would be better if we were using `mockState.metamask.keyringsMetadata[0]` here (as always, assuming the 1st one being the primary keyring)